frontend/app-development/layout/SettingsModalButton/SettingsModal/utils/tabUtils/setupTabUtils/setupTabUtils.ts

```diff
@@ -0,0 +1,15 @@
+import { isDateAfter } from 'app-development/utils/dateUtils';
+
+/**
+ * Returns if two dates are valid or not, to be used to display the error
+ * message in the SetupTab.
+ *
+ * @param from the date string from
+ * @param to the date string to
+ *
+ * @returns true if it is valid,false if it is not valid.
+ */
+export const getIsDatesValid = (from: string, to: string): boolean => {
+  if (from === undefined || to === undefined) return true;
```

Comment:
Ref. en annen kommentar, så må vi huske å oppdatere denne metoden til å sjekke på tom string og ikke undefined. 😊

Reply:
Det er jeg litt uskker på om vi bør eller ikke. Kan godt legge på en sjekk for tom string også, men som du ser i bildet under så brukes den på direkte verdien fra objektet, og den kan være undefined der. 
![Screenshot 2023-10-20 at 12 34 25](https://github.com/Altinn/altinn-studio/assets/133344438/ec2e4d75-c861-417e-8cd9-28bb2d1778e4)

Reply:
Legger på sjekk på tom string og undefined